scripts/native_arm64.proj

```diff
@@ -0,0 +1,63 @@
+<Project DefaultTargets="Build">
```

Comment:
...and this to `native.arm64.proj`

Reply:
OK

Reply:
I don't think it's possible to run multiple build pipelines on a free account. Aren't we able to build both of distros in one pipeline?

> Is there a way to change the `dotnet publish` command to produce the portable `zip` distribution only when building for Arm64?

The publishing targets are here: https://github.com/gitextensions/gitextensions/blob/master/GitExtensions/Project.Publish.targets#L271. I don't think it should be too difficult to condition `CreateMsi` target based on a MSBuild property.

Reply:
> I don't think it's possible to run multiple build pipelines on a free account. Aren't we able to build both of distros in one pipeline?

I'm out of my depth here, so I can't comment any further.

> The publishing targets are here: https://github.com/gitextensions/gitextensions/blob/master/GitExtensions/Project.Publish.targets#L271. I don't think it should be too difficult to condition `CreateMsi` target based on a MSBuild property.

Yes, it's trivial to add a condition to the `CreateMsi` target based on a property, but deciding on *which* property to use is tricky. I've tried various existing properties like `ProcessorArchitecture`, `RuntimeIdentifier`, `Platform` by adding extra options such as `-a ARM64` or `-r win-arm64` or `--use-current-runtime` to the `dotnet publish` command, but they have various side-effects in the resulting package. Like, using the `-r win-arm64` or `--use-current-runtime` on WoA would produce a package for Arm64 having similar problems as mentioned in #9990.

Eventually I decided to define my own property `Arm64Build`, so that to produce the x86/x64 distributions the original command would still work as before:
```
dotnet publish
```
while to produce Arm64 package would require setting the `Arm64Build` flag so that it would skip the `CreateMsi` step:
```
dotnet publish /p:Arm64Build=true
```
In summary, here are the complete (unchanged) commands to build GE for x86/x64 platform:
```
dotnet build
dotnet build .\scripts\native.proj
dotnet publish
```
while to build GE for WoA:
```
dotnet build
dotnet build .\scripts\native.proj /p:Arm64Build=true
dotnet publish /p:Arm64Build=true
```
Ideally, the first set of commands should have worked for both situations, without defining the new `Arm64Build` property, but I haven't found any existing MSBuild property up to the task. Any help/suggestion is quite welcome.